README.md

```diff
@@ -21,11 +21,9 @@ application to capture and report distributed traces to Splunk APM.
 This Splunk distribution comes with the following defaults:
 
 - [W3C tracecontext](https://www.w3.org/TR/trace-context/) and [W3C baggage](https://www.w3.org/TR/baggage/) context propagation.
-- [Jaeger thrift
-  exporter](https://opentelemetry-python.readthedocs.io/en/stable/exporter/jaeger/jaeger.html)
-  configured to send spans to a locally running [SignalFx Smart
-  Agent](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-smart-agent.html)
-  (`http://localhost:9080/v1/trace`).
+- [OTLP gRPC exporter](https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html)
+  configured to send spans to a locally running [OpenTelemetry Collector](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html)
+  (`http://localhost:4317`).
```

Comment:
splunk-opentelemetry will have OTLP gRPC as the default configured exporter. As in if no config is provided, it'll try to use OTLP gRPC.

Package management and dependencies are a bit different though. splunk-opentelemetry has 3 optional dependencies: otlp exporter, jaeger exporter and b3 exporter. If one installs `pip install splunk-opentelemetry`, it will not install any of these libraries. Users need to know which components they need and install them accordingly by as `pip install splunk-opentelemetry[otlp,jaeger,b3]`.

In order to make it easy to install everything, we also have an "all" option that pulls in everything which can be used as `pip install splunk-opentelemetry[all]`. This is what we document and recommend everywhere except in advanced sections where we are specifically explaining how to install a subset of libs. So as long as users follow the docs here or Splunk product docs, everything should be installed by default  and OTLP gRPC should be used by the library out of the box.